### DISCUSSION TITLE
账号密码在哪里？登录失败，注册也失败

### DISCUSSION THREAD

**Discussion by 000000000111111111** [illacloud/illa-builder#1055] [中文支持 Chinese Support]
windows下安装
> illa deploy --self --port=10000
可以访问、
登录提示，登录失败
注册也提示，注册失败
是个啥情况？

**Comment by 000000000111111111** [comment]
http://localhost/supervisor/api/v1/auth/signup
浏览器抓包发现这个访问503 
直接访问提示
upstream connect error or disconnect/reset before headers. reset reason: connection failure

**Comment by karminski** [comment]
@Chanli52580 @sheiruan 您好，请运行一下 docker logs illa_builder 然后把输出粘贴一下？我们会帮您看一下。
